packages/fx-core/templates/plugins/resource/spfx/solution/README.md

```diff
@@ -15,7 +15,7 @@ The SharePoint Framework (SPFx) is a page and web part model that provides full
 
 ## Prerequisites
 
-> - Node.js v12/14/16
+> - Node.js v14/16
```

Comment:
Don't forget to update the sample code
https://github.com/OfficeDev/TeamsFx-Samples/blob/e9c78bfc1ebe61213837782caf4276f7c5ee8c36/todo-list-SPFx/README.md?plain=1#L21

Reply:
PR raised: https://github.com/OfficeDev/TeamsFx-Samples/pull/459. Thanks!